### PR TITLE
fix(features): remove text-decoration on features

### LINKED ISSUE #381: underlining of titles of features
# Bug Report 🐛
The titles of the features blocks like : 'Concerto' are underlined upon hovering.

## Expected Behavior
Since two of the titles: 'CICERO' and 'ERGO' are actually images, there is no underlining upon hovering. The same should happen with 'Concerto' and the other 2.

## Current Behavior
The other 2 along with 'Concerto' get underlined.

## Steps to Reproduce

1.Open the link : https://docs.accordproject.org/
2. hover over the features' titles.

### Desktop
 - OS: [e.g. macOS]  Ubuntu 20.04.2 LTS
 - Browser: [e.g. Chrome, Safari] Firefox 
 - Version: [e.g. 0.22.15] 91.0.2 (64-bit)

## Detailed Description

The anchor tags already have 'text-decoration: none' but it isn't working. I can fix this.

### PR DESCRIPTION
**Pull request by shubhangi013**
Signed-off-by: shubhangi013 <choudharyshubhangi13@gmail.com>

# Closes #381 

The PR aims at removing the underlining effect that causes a bad-user experience because of being uneven. 
### Changes

- <ONE> added 'text-decoration: none!important;'

### Related Issues
- Issue #381 

### Author Checklist
- [x] Ensure you provide a [DCO sign-off](https://github.com/probot/dco#how-it-works) for your commits using the `--signoff` option of git commit.
- [ ] Vital features and changes captured in unit and/or integration tests
- [x] Commits messages follow [AP format](https://github.com/accordproject/techdocs/blob/master/DEVELOPERS.md#commit-message-format)
- [ ] Extend the documentation, if necessary
- [x] Merging to `master` from `fork:branchname`
- [ ] Manual accessibility test performed
    - [ ] Keyboard-only access, including forms
    - [ ] Contrast at least WCAG Level A
    - [ ] Appropriate labels, alt text, and instructions

### CODE REVIEW

**Comment by Michael-Grover** [comment]
@shubhangi013 this looks good to me. It looks like some CSS has changed, so I think it would be good to have an engineer take and confirm only the intended links are affected before merging. @DianaLease @irmerk

**Review by irmerk — CHANGES_REQUESTED** [review]
I think the `website/package-lock.json` changes can be discarded.

**Comment by shubhangi013** [comment]
> I think the `website/package-lock.json` changes can be discarded.

Oh alright ! I'll do that :)

**Comment by shubhangi013** [comment]
Sorry for this uneven commit history, my VSCode was acting up :sweat_smile: 
Could you review it once? @irmerk

**Comment by shubhangi013** [comment]
@Michael-Grover could you have a look at it please 😅

**Comment by Michael-Grover** [comment]
@shubhangi013 it looks good to me, I'm not an engineer though, so I still think it would be good to have @DianaLease or @irmerk give the final approval.

**Comment by shubhangi013** [comment]
> @shubhangi013 it looks good to me, I'm not an engineer though, so I still think it would be good to have @DianaLease or @irmerk give the final approval.

Ah right! Thankyou :)

**Comment by irmerk** [comment]
You should rebase this down to one commit and correct the DCO signoff.

**Comment by shubhangi013** [comment]
> You should rebase this down to one commit and correct the DCO signoff.

done!

**Comment by irmerk** [comment]
We need to check if altering `a` tags in this PR doesn't have any unintended effects on the rest of the site.

**Comment by shubhangi013** [comment]
> We need to check if altering `a` tags in this PR doesn't have any unintended effects on the rest of the site.

Oh right. I did run it locally tho and they worked fine. 😅

**Review by irmerk — APPROVED** [review]
Looks good from my manual review. Would like two other manual reviews from @accordproject/maintainers members.

**Review by mttrbrts — APPROVED** [review]

**Review by dselman — APPROVED** [review]

**Pull request merged**
